src/main/java/org/httpserver/config/ConfigurationManager.java

```diff
@@ -0,0 +1,66 @@
+package org.httpserver.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.httpserver.util.Json;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ConfigurationManager {
+    private static ConfigurationManager myConfigurationManager;
+    private static Configuration myCurrentConfiguration;
+    private ConfigurationManager() {
+    }
+    public static ConfigurationManager getInstance(){
```

Comment:
Singleton pattern - is not really what you would want to use here. This class has some state (it stores the instance of `Configuration`), but it feels redundant here.